support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -35,8 +38,10 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
     services: Services
   ): FutureHandlerResult = {
     val now = () => OffsetDateTime.now
+    val now2 = () => LocalDate.now(DateTimeZone.UTC)
```

Comment:
please explain!

Reply:
sorry forgot to add the explanation here - it's because the existing `now` is a java one, wheras the one we use is a joda one.  Technically joda is not needed any more as the functionality is in the java library now, but I wasn't sure whether I should change the new one to use the java time, or the old one to use a joda time, or neither.  Could be one for a health task one way or the other.